File: AlbumLibrary/src/main/java/com/album/listener/AlbumCameraListener.java
package com.album.listener;

import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;

/**
 * by y on 28/08/2017.
 * <p>
 * customize camera ui
 */

public interface AlbumCameraListener {

    /**
     * 自定义相机
     *
     * @param fragment {@link com.album.ui.fragment.AlbumFragment}
     */
    void startCamera(@NonNull Fragment fragment);
}

File: AlbumLibrary/src/main/java/com/album/listener/AlbumCameraListener.kt
package com.album.listener

import com.album.ui.fragment.AlbumBaseFragment

/**
 * by y on 28/08/2017.
 *
 *
 * customize camera ui
 */

interface AlbumCameraListener {

    /**
     * 自定义相机
     *
     * @param fragment [com.album.ui.fragment.AlbumFragment]
     */
    fun startCamera(fragment: AlbumBaseFragment)
}
